src/content/reference/manifest.mdx

```diff
@@ -972,6 +972,6 @@ $ cat okteto.yaml
 ...
 ```
 
-When you any Okteto CLI command, `image` will be `node:v1.5`.
```

Comment:
Could you review if this is missing also in versioned versions and change it also there?

Reply:
Done!